layouts/default.slim

```diff
@@ -3,9 +3,13 @@ html
   head
     == render '/header.*'
   body lang == (@item[:lang] || 'cn')
-    #content
-      == render '/menu.*'
-      == yield
+    .pusher
```

Comment:
I think it's better to create a new layout and nanoc rules for kitchen-sink page, since not all pages need a sidebar. It's OK for this PR, I'll do the later work to refactoring layout.